src/Proxy/DriveItemProxy.php

```diff
@@ -441,4 +484,4 @@ public function copy(self $destinationItem, array $options = [])
 
         return $headers['Location'][0];
     }
-}
+}
```

Comment:
Final line break is also part of the coding style.

Reply:
ok